schema.sql

```diff
@@ -39,5 +39,13 @@ CREATE TABLE coin_transactions (
        status text NOT NULL,
 
        created_time timestamptz NOT NULL DEFAULT now()
-)
+);
 
+CREATE MATERIALIZED VIEW statistics AS (
```

Comment:
Eventually this is going to take a long time. The actual ideal solution here is to just have an atomic stats table that isn't generated from a query / as a view, so that fetching stats is `O(1)`, or I guess `O(2)` if you consider the `UPDATE`s you have to do. Or just update playing catch-up periodically since stats aren't important. But this is probably fine for now since this won't be used often anyways. You can prolong this current method by increasing your TTL, probably up to 10 minutes, since no ones cares / will know.

Just food for thought.

Reply:
Hm. Thanks for taking the time!